test/pummel/test-net-timeout.js

```diff
@@ -79,7 +79,7 @@ process.on('exit', function() {
   assert.ok(starttime != null);
   assert.ok(timeouttime != null);
 
-  diff = timeouttime - starttime;
+  var diff = timeouttime - starttime;
```

Comment:
Strange indeed. If I run this test manually on master without the framework, I see a ReferenceError. Not too familar with pummel tests, how are they executed?